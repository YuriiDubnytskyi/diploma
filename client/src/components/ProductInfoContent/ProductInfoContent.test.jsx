import React from "react";
import ProductInfoContent from "./ProductInfoContent";

const setUp = (props) => shallow(<ProductInfoContent {...props} />);

describe("Test HTML in ProductInfoContent with props", () => {
    it("Should contain info", () => {
        const component = setUp({
            data: {
                images: [],
                name: "Nokia",
                price: "122",
                shortInfo: "Nokia phone",
                producer: "Ukraine",
                info: "Опис",
                properties: [{ property: "Память", value: "125" }],
            },
        });
        expect(component).toMatchSnapshot();
    });
});
