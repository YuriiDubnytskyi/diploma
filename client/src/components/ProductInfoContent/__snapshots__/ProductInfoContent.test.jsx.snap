// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test HTML in ProductInfoContent with props Should contain info 1`] = `
<div
  className="product__container product-container "
>
  <div
    className="product-img"
  >
    <lazy
      images={Array []}
    />
  </div>
  <div
    className="product-short-info"
  >
    <h2
      className="product-short-info-title"
    >
      Nokia
    </h2>
    <p
      className="product-short-info-description"
    >
      Nokia phone
    </p>
    <hr
      className="product-short-info-line"
    />
    <p
      className="product-short-info-price"
    >
      Ціна - 
      122
    </p>
    <hr
      className="product-short-info-line"
    />
    <div
      className="product-short-info-btn"
    >
      <p
        className="product-short-info-buy"
        onClick={[Function]}
      >
        <i
          className="fa fa-shopping-bag"
        />
        додати до кошика
      </p>
      <p
        className="product-short-info-like"
        onClick={[Function]}
      >
        <i
          className="fa fa-heart"
        />
      </p>
    </div>
    <hr
      className="product-short-info-line"
    />
    <p
      className="product-short-info-producer"
    >
      Країна виробник -- 
      Ukraine
    </p>
  </div>
  <div
    className="product__info"
  >
    <div
      className="product-info"
    >
      <h3
        className="product-info-title"
      >
        Опис Товару
      </h3>
      <p
        className="product-info-description"
      >
        Опис
      </p>
    </div>
    <div
      className="product-properties"
    >
      <h3
        className="product-info-properties"
      >
        Характеристики
      </h3>
      <div
        className="product-properties__container"
      >
        <div
          className="product-properties-item"
          key="0"
        >
          <p
            className="item-property"
          >
            Память
          </p>
          <p
            className="item-value"
          >
            125
          </p>
        </div>
      </div>
    </div>
  </div>
</div>
`;
